jekyll/_cci2/workflows.md

```diff
@@ -300,62 +300,47 @@ workflows:
 
 In the example, `filters` is set with the `branches` key and the `only` key with the branch name. Any branches that match the value of `only` will run the job. Branches matching the value of `ignore` will not run the job. See the [Sample Sequential Workflow config with Branching](https://github.com/CircleCI-Public/circleci-demo-workflows/blob/sequential-branch-filter/.circleci/config.yml) for a full example.
 
-### Git Tag Job Execution
+### Executing Jobs for a Git Tag
 
-CircleCI treats tag and branch filters differently when deciding whether a job should run.
+CircleCI does not run jobs for tags
+unless you explicitly specify tag filters.
+Additionally,
+if a job requires any other jobs (directly or indirectly),
+you must specify tag filters for those jobs.
 
-1. For a branch push unaffected by any filters, CircleCI runs the job.
-2. For a tag push unaffected by any filters, CircleCI skips the job.
+In the example below,
+two workflows are defined:
 
-Item two above means that a job **must** have a `filters` `tags` section to run as a part of a tag push and all its transitively dependent jobs **must** also have a `filters` `tags` section. 
+- `untagged-build` runs the `build` job for all branches.
+- `tagged-build` runs `build` for all branches **and** all tags starting with `v`.
 
-Following is a very basic example for building any branch and using tags. The regular expression is a full match rather than a partial match. For example, `only: /^config-test.*/` matches any tag with the prefix `config-test-111` and `only: /^config-test/` matches all tags that match `config-test`.  To match the common use case of a semantic versioning, for example, use `/version-2\.1\.[3-7]/` to match `version-2.1.`(3 through 7).
```

Comment:
we should keep this info about semantic versioning, it can be in a sub-section with a heading, but this question was coming up, so we added it

Reply:
It looks like we've got a section heading called "Regular Expression Support" immediately after the section for tag filters. Good candidate for this section?

Reply:
Yes, I thought there was one! 👍